udata/frontend/markdown.py

```diff
@@ -39,14 +40,30 @@ def nofollow_callback(attrs, new=False):
     That callback is not splitted in order to parse the URL only once.
     """
 
-    if (None, u"href") not in attrs:
+    allow_mailto = current_app.config['MD_ALLOW_MAILTO']
+
+    if (None, "href") not in attrs :
         return attrs
+
     parsed_url = urlparse(attrs[(None, 'href')])
+
+    if parsed_url.scheme == "mailto" :
+        scheme = 'mailto'
+        joiner = ':'
+    else :
+        scheme='https' if request.is_secure else 'http'
+        joiner = '://'
+
     if parsed_url.netloc in ('', current_app.config['SERVER_NAME']):
-        attrs[(None, 'href')] = '{scheme}://{netloc}{path}'.format(
-            scheme='https' if request.is_secure else 'http',
-            netloc=current_app.config['SERVER_NAME'],
-            path=parsed_url.path)
+        netloc_override = current_app.config['SERVER_NAME']
+        if parsed_url.scheme == "mailto":
+          netloc_override = ''
+          if not allow_mailto:
+            #attrs[(None, 'href')] = ''
```

Comment:
remove

Reply:
if removed the tests won't pass in the case `MD_ALLOW_MAILTO == True` (but will in the case `MD_ALLOW_MAILTO == False`

Reply:
if you remove a comment?!

Reply:
aaaah sorry I though you wanted to remove the whole block !!

Reply:
I removed the comment :))